crates/core/Cargo.toml

```diff
@@ -55,6 +64,8 @@ url.workspace = true
 indicatif.workspace = true
 tabled.workspace = true
 
+ruint = { version = "1.12.3", default-features = false, features = ["alloc"]}
```

Comment:
Is this needed? Pretty sure all ruint primitives should be available through alloy

And, if needed, move version to root Cargo.toml